docs/userManual.md

```diff
@@ -287,18 +287,52 @@ marked with x.
 ### Data schema views
 
 The data schema views allow users to browse the schema of the current
-table and select a set of columns from the dataset to focus on.
+table and select a set of columns from the dataset to focus on. This
+feature is especially useful when the table contains too many columns
+to display at once, and the user wants to focus on a subset of them.
 
 The following example shows a schema view; the rows in a schema view
 are the description of the columns of the data table.  In this example
 there are three rows selected.
 
 ![Schema view](schema-view.png)
 
-Once the user selects a set of columns, the user can display a view of the
-data table restricted to the selected columns using the View/Selected
-columns menu.
 
+There are two ways to modify the selection:
+1. By using the mouse: Clicking on a column description toggles its
```

Comment:
The user manual already has a section describing how mouse-based selection works. I would rather point to that section here than write it again. It will also make it easier to change behaviors if we decide we don't like the current state-machine.